### ISSUE TITLE
About VAE

### ISSUE THREAD

**Issue by zhgqcn** [Virtsionis/torch-nilm#12]
![65c190571e1bbd8d528926c4d80e7a7](https://user-images.githubusercontent.com/44897668/178105777-d3c558e8-a29a-4010-a7d5-ec4d2c481fdf.png)
![6e6bce50563cb76fcc1236c0b0a6bf5](https://user-images.githubusercontent.com/44897668/178105806-aab97abd-dd78-44ca-9ea6-c624c9c6a0e5.png)

the kl loss is always the same, so the mu and std.

**Comment by Virtsionis** [comment]
please give me your complete settings to reproduce,,, also tell me the lines that you put your logs

**Comment by Virtsionis** [comment]
Running a fast check I noticed the following: 
1) I put the following logs in lab/training_tools.py inside training_step method of VIBTrainingTools class
        self.log('class_loss', class_loss, prog_bar=True)
        self.log('info_loss', info_loss, prog_bar=True)
and I can see the losses reducing.

2) In terms of the mu, it seems that it stays as a tensor filled with zeros between training steps in the first epoch. Did you trained for more than one epoch?
3) std changes as expected, I can't reproduce your numbers.

Maybe I missed something in the original implementation. I will look in to this and comeback with an answer.

As it seems VAEs are hot topic, so I am planning on adapting a library that contains lots of implementation in to 
Torch - NILM in the near futur.
